src/smt/theory_lra.cpp

```diff
@@ -1395,10 +1395,22 @@ class theory_lra::imp {
 #endif
         }
         else {
-            literal div_ge_0   = mk_literal(a.mk_ge(div, zero));
-            literal div_le_0   = mk_literal(a.mk_le(div, zero));
-            literal p_ge_0     = mk_literal(a.mk_ge(p, zero));
-            literal p_le_0     = mk_literal(a.mk_le(p, zero));
+#if 0
```

Comment:
seems these literals shouldn't be created?

Reply:
I am not sure. These literals used in code below which is surrounded by #if 0 ... #endif